fendermint/vm/topdown/src/sync/syncer.rs

```diff
@@ -266,6 +268,44 @@ where
         Ok(data.0)
     }
 
+    #[instrument(skip(self))]
+    async fn check_data(
+        &self,
+        height: BlockHeight,
+        current_view: &ParentViewPayload,
+        parent_block_hash: &[u8],
```

Comment:
Isn't there an alias for block hash at least?

Reply:
updated